src/common/metrics/hospitalizations.tsx

```diff
@@ -158,6 +158,7 @@ function renderDisclaimer(projections: Projections): React.ReactElement {
       <ExternalLink href="/covid-risk-levels-metrics#icu-capacity-used">
         ICU capacity methodology
       </ExternalLink>
+      .
```

Comment:
If you were so inclined, we can probably drop the ICU headroom reference above now. 😁

Reply:
👍